t/40-script_load_templates.t

```diff
@@ -58,22 +58,16 @@ is $ret, 255, 'error because of insufficient permissions';
 $apikey    = 'ARTHURKEY01';
 $apisecret = 'EXCALIBUR';
 $args      = "--host $host --apikey $apikey --apisecret $apisecret $filename";
-combined_like sub { $ret = run_once($args); }, qr/Job group.+not found/, 'Invalid job group';
```

Comment:
It looks we removed scenario when 'Job group not found' is returned. (just observation)

Reply:
Right. This is because we do handle the case of creating the group now.